conformance-checking/src/test/java/org/archcnl/conformancechecking/impl/ConformanceCheckImplTest.java

```diff
@@ -83,16 +84,16 @@ public void testCoarse() throws FileNotFoundException {
                 ConformanceCheckOntologyClassesAndProperties.get(
                         ConformanceCheckDatatypeProperties.hasCheckingDate, expected);
         expected.getProperty(
-                        ConformanceCheckOntologyClassesAndProperties.getConformanceCheckIndividual(
-                                expected),
+                        ConformanceCheckOntologyClassesAndProperties.createIndividual(
+                                ConformanceCheckOntClasses.ConformanceCheck, expected),
                         hasCheckingDate)
                 .remove();
         hasCheckingDate =
                 ConformanceCheckOntologyClassesAndProperties.get(
                         ConformanceCheckDatatypeProperties.hasCheckingDate, actual);
         actual.getProperty(
-                        ConformanceCheckOntologyClassesAndProperties.getConformanceCheckIndividual(
-                                actual),
+                        ConformanceCheckOntologyClassesAndProperties.createIndividual(
+                                ConformanceCheckOntClasses.ConformanceCheck, actual),
                         hasCheckingDate)
                 .remove();
 
```

Comment:
We might want to add test cases ensuring the correctness of the getId() method as the state statement is error-prone. But that is no longer part of issue #128 so not it is not your responsibility.